src/documents/IoCReferences.md

```diff
@@ -160,12 +160,12 @@ fluid.defaults("fluid.prefs.enactor.tableOfContents", {
                     container: "{toc}.dom.tocContainer"
                 }
             }
-        }  
+        }
     }
 });
 ```
 
-## Examples of `{<componentRef>}.<path to member>` ##
+## Examples of {<componentRef>}.<path to member> ##
```

Comment:
This doesn't render properly - the angle brackets and their content are stripped.